src/vos/tests/evt_ctl.c

```diff
@@ -2438,13 +2365,25 @@ main(int argc, char **argv)
 		goto out;
 	}
 
-	while ((rc = getopt_long(argc, argv, "C:a:m:e:f:g:d:b:Docl::ts:",
-				 ts_ops, NULL)) != -1) {
-		rc = ts_cmd_run(rc, optarg);
-		if (rc != 0)
-			goto out;
+	/* First, execute Internal tests in the command */
+	for(j=0; j<argc;j++){
```

Comment:
(style)  spaces required around that '=' (ctx:VxV)
(style)  spaces required around that '<' (ctx:VxV)
(style)  space required after that ';' (ctx:VxV)
(style)  space required before the open brace '{'
(style)  space required before the open parenthesis '('